docs/docker.md

```diff
@@ -0,0 +1,71 @@
+## Horovod in Docker
+
+To ease the installation process on GPU machines, we have published the reference [Dockerfile](../Dockerfile) that
```

Comment:
ease --> streamline
"we have published the reference [Dockerfile](../Dockerfile) that" --> "we published the reference [Dockerfile](../Dockerfile) so you can get started with Horovod in minutes."